### DISCUSSION TITLE
New alert folders show (empty) in dashboards view

### DISCUSSION THREAD

**Discussion by jgournet** [grafana/grafana#66810] [Alerting]
Hi,

Not sure if it's a feature or if I'm doing something wrong:
When I create a new alert, it forces me to assign it to a folder - fine: I'll create a new alert (?) folder.

But this shows up as an empty dashboard folder in dashboard "Browse" ... why ?

Note: ok, I could just assign my alert into an existing dashboard folder to hide this ... but sadly, it's not "that" simple, since I use provisioning via K8S config maps from different sources: when doing so, I found that you cannot have several times the same "folder":
```
{
  "apiVersion": 1,
  "groups": [
    {
      "orgId": 1,
      "name": "My Alerts",
      "folder": "My Alerts",    <= this will not work
      "interval": "1m",
      "rules": [
        {
```

Am I doing something wrong at some point ? Or is there a workaround ?

Thanks for your help

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
